templates/shared/_blogsection.html

```diff
@@ -0,0 +1,69 @@
+{# djlint: off #}
+<noscript>
+  <section class="{% if section_classes %}{{ section_classes }}{% else %}p-strip{% endif %}">
+    <div class="row">
+      <h3><a href="/blog{% if tag_name %}/tag/{{ tag_name }}{% endif %}">Read the latest news on our blog&nbsp;&rsaquo;</a></h3>
+    </div>
+  </section>
+</noscript>
+
+<section class="u-hide {% if section_classes %}{{ section_classes }}{% else %}p-strip{% endif %}" data-js="latest-news">
```

Comment:
```suggestion
<section class="u-hide {% if section_classes %}{{ section_classes }}{% else %}p-strip{% endif %}">
```
Is this needed? Doesn't look like it's used anywhere. If it is a JS hook, our standard is to use a class like 'js-latest-news'. [Data attributes](https://developer.mozilla.org/en-US/docs/Learn_web_development/Howto/Solve_HTML_problems/Use_data_attributes) should be used for data

Reply:
Hey, it's being used in latest-news package, it could be better used as a class but then this has to be changed in the package first.
![image](https://github.com/user-attachments/assets/b449ef16-245d-4381-91ed-66dbeecace53)